lib/importers/article_importer.rb

```diff
@@ -2,27 +2,33 @@
 
 #= Imports articles from Wikipedia into the dashboard database
 class ArticleImporter
-  def self.import_articles(ids)
-    article_ids = ids.map { |id| { 'id' => id } }
+  def initialize(wiki)
+    @wiki = wiki
+  end
+
+  def import_articles(ids)
+    article_ids = ids.map { |id| { 'mw_page_id' => id } }
     articles_data = []
     article_ids.each_slice(40) do |some_article_ids|
-      articles_data += Replica.get_existing_articles_by_id some_article_ids
+      articles_data += Replica.new(@wiki).get_existing_articles_by_id some_article_ids
     end
     return if articles_data.empty?
     articles = []
     articles_data.each do |article_data|
       articles << Article.new(id: article_data['page_id'],
+                              mw_page_id: article_data['page_id'],
```

Comment:
Refers to article_data more than self (maybe move it to another class?) (FeatureEnvy)